versioned_docs/version-v6/vue/quickstart.md

```diff
@@ -125,24 +116,15 @@ From there, let's look at the template.
 
 All Vue components must have a `<template>`. Inside of there, we place our `IonApp` and `IonRouterOutlet` components.
 
-Finally, let's look at the component definition:
+Finally, let's look at the script:
 
 ```tsx
+<script setup lang="ts">
 import { IonApp, IonRouterOutlet } from '@ionic/vue';
-import { defineComponent } from 'vue';
-
-export default defineComponent({
-  name: 'App',
-  components: {
-    IonApp,
-    IonRouterOutlet,
-  },
-});
+</script>
 ```
 
-Vue 3 offers a new `defineComponent` function when creating components for improved tooling support, and we are going to use that here. We first define the name of the component, and then we supply the components that we will use in our template.
-
-There are several arguments you can supply here such as `methods`, `setup` and more. This is explained as part of Vue's [Composition API Documentation](https://v3.vuejs.org/guide/composition-api-introduction.html#why-composition-api).
+We are going to use Vue's [`script setup`](https://vuejs.org/api/sfc-script-setup.html) syntax when creating components.
```

Comment:
I'd probably add something that notes how we are importing the IonApp and IonRouterOutlet components, and since we are using script setup the template has access to those components.